www/RogerthatPlugin.js

```diff
@@ -293,10 +293,12 @@ RogerthatPlugin.prototype.features = {
     callback: undefined
 };
 
-RogerthatPlugin.prototype.getBadges = function(){
-    return new Promise(function(resolve, reject){
-        utils.exec(resolve, reject, "getBadges", []);
-    });
+RogerthatPlugin.prototype.badges = {
```

Comment:
needs to stay getBadges otherwise you have to modify 3 or 4 embedded apps that are already made atm

Reply:
then update those embedded apps

Reply:
why? rogerthat.getBadges is simpler